### ISSUE TITLE
Design Intent - Accessibility Feedback - Login.gov Adoption, VA Profile, Sign in Page

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#63562]
## VFS actions
- [x] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [x] Comment on this ticket:
  - [x] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [x] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [x] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [x] Close the ticket when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Conditional Must:** If you end up going the route of the standard alert at the top of the page, make sure you're paying attention to the heading structure. Standard alerts have a heading, which means everything that comes after that alert sits "inside" the section labeled by that heading until you reach the next heading. It may make sense to include extra headings for the preferred and less-preferred login methods (either together as a group or separately as two groups). 
   - Note that adding some heading structure separating the two groups of login methods may also help to reinforce the message about preferring Login.gov and ID.me --- and will communicate that difference to screen reader users. Although the changes to the DSLogon and MyHealtheVet actions de-emphasizes them visually, visual emphasis isn't communicated to screen reader users.
- **Should:** If we're being semantically honest, all of the login methods should be links and not buttons. To the extent that it's possible, I'd love to see the Login.gov and ID.me actions redesigned as links --- maybe some variation on [action links](https://design.va.gov/components/link/action) with icons and branding consistent with those authentication services. That would probably need to go through the Design System Council but I think if you've got something that looks clean and consistent (and is only used in this one place) people will probably be supportive.
   
   If that's not practical, I think it's probably fine to have two methods as buttons and two methods as links. Having those two element types in use for one kind of interaction makes me uncomfortable, since assistive technologies often treat buttons and links differently. But users will typically only be using one login method at a time, and any potential confusion for assistive technology users around the two semantic element types will be resolved after the first time they log in. And since logging in is probably something you only do once per session, it's not a persistent opportunity for confusion that you'll encounter multiple times as you use VA.gov.
   
   So, short version: These should all be links and I would love for you to solve that design challenge! But I wouldn't flag it as an issue at launch, unless I get feedback from some of the other accessibility folks I've pinged to make sure I'm not missing something important.

**Comment by MOStudioCarolyn** [comment]
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbU1IIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--17d223e346422759182f0ec85346c5cf849efb99/Screen%20Shot%202023-08-15%20at%2011.49.48%20AM.png)
Must and Should feedback incorporated
